### PR TITLE
whitelisted_ips configuration should be specific

### PR DESCRIPTION
**Pull request by frenesim**
whitelisted_ips should be in the a dedicated environment configuration file.

### CODE REVIEW

**Comment by gsamokovarov** [comment]
I think we can put this statement for the whole config. Can you add a question/answer to the FAQ section?

File: README.markdown [hunk 1]
```diff
@@ -108,10 +108,10 @@ By default, only requests coming from IPv4 and IPv6 localhosts are allowed.
 the console.
 
 You can whitelist single IP's or whole networks. Say you want to share your
-console with `192.168.0.100`. You can do this:
+console with `192.168.0.100`. You can do this on your development environment:
```
Comment (gsamokovarov) [inline README.markdown]: Can you drop this? See the main comment.

File: README.markdown [hunk 2]
```diff
@@ -108,10 +108,10 @@ By default, only requests coming from IPv4 and IPv6 localhosts are allowed.
...
 
 ```ruby
-class Application < Rails::Application
+Rails.application.configure do
```
Comment (gsamokovarov) [inline README.markdown]: Can you change this for all `class Application < Rails::Application` examples?

Comment (frenesim) [inline README.markdown]: sure. I'll try.

File: README.markdown (hunk 1 above)
Comment (frenesim) [inline README.markdown]: ok

File: README.markdown (hunk 2 above)
Comment (gsamokovarov) [inline README.markdown]: Thanks!

File: README.markdown [hunk 3]
```diff
@@ -193,6 +193,15 @@ class Application < Rails::Application
 end
 ```
 
+### I'm getting "undefined method 'web_console'"
+
+Configuration should be set in a dedicated environment, not in the
+application.rb.
+Move your web-console configuration to the files under config/environment.
```
Comment (gsamokovarov) [inline README.markdown]: Can you mention `config/environments/development.rb` explicitly. This is the configuration file we want to advertise our configuration for.

Comment (frenesim) [inline README.markdown]: sure. It looks better in fact

File: README.markdown [hunk 4]
```diff
@@ -193,6 +193,15 @@ class Application < Rails::Application
 end
 ```
 
+### I'm getting "undefined method 'web_console'"
+
+Configuration should be set in a dedicated environment, not in the
```
Comment (gsamokovarov) [inline README.markdown]: Can we put it all in one paragraph like:

```
Make sure Web Console configuration lives in `config/environments/development.rb`
and not in `config/application.rb`.
```

File: README.markdown [hunk 5]
```diff
@@ -193,6 +193,15 @@ class Application < Rails::Application
 end
 ```
 
+### I'm getting "undefined method 'web_console'"
```
Comment (gsamokovarov) [inline README.markdown]: Can we phrase this as a question? Also can we drop the quoting. Something like:

```Why I'm getting an undefined method `web_console` during boot?```

**Comment by gsamokovarov** [comment]
@frenesim Sory for the late response. Can you rebase and squash those commits?

**Comment by frenesim** [comment]
I will do

**Comment by gsamokovarov** [comment]
@frenesim ping :smile_cat:

**Comment by gsamokovarov** [comment]
I'll introduce those changes before release. Thank you for bringing the topic up and helping us out with the docs. I'll make sure I credit you in the changelog.

**Pull request closed without merging**